contao/templates/js_glightbox_caption.html5

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Contao\Template;
+
+$GLOBALS['TL_CSS']['glightbox'] = 'bundles/contaoglightbox/css/glightbox.min.css|static';
+echo Template::generateScriptTag('bundles/contaoglightbox/js/glightbox.min.js', false, null);
+
+?>
+<script>
+(function(){
+  'use strict';
+  document.querySelectorAll('a[data-lightbox]').forEach((element) => {
+    if (!!element.dataset.lightbox) {
+      element.setAttribute('data-gallery', element.dataset.lightbox);
+      const caption = element.nextElementSibling;
```

Comment:
This wouldn't be very stable, in case anyone adjusted their templates and has something else there. Maybe more something like `element.closest('figure')?.querySelector('figcaption')`?